packages/create-app/templates/default-app/packages/backend/src/plugins/search.ts.hbs

```diff
@@ -54,8 +53,6 @@ export default async function createPlugin(
   const { scheduler } = await indexBuilder.build();
   scheduler.start();
 
-  useHotCleanup(module, () => scheduler.stop());
```

Comment:
same for this one, let's keep it for now right?

Reply:
Done 👍